rfc/5/versions/1/index.md

```diff
@@ -0,0 +1,977 @@
+
+
+### "coordinateTransformations" metadata
+
+"coordinateTransformations" describe the mapping between two coordinate systems (defined by "axes").
+For example, to map an array's discrete coordinate system to its corresponding physical coordinates.
+Coordinate transforms are in the "forward" direction.
+They represent functions from *points* in the input space to *points* in the output space. 
+
+- MUST contain the field "type" (string).
+- MUST contain any other fields required by the given "type" (see table below).
+- MUST contain the field "output" (string),
+  unless part of a `sequence` or `inverseOf` (see details).
+- MUST contain the field "input" (string),
+  unless part of a `sequence` or `inverseOf` (see details).
+- MAY contain the field "name" (string).
+  Its value MUST be unique across all "name" fields for coordinate transformations.
+- Parameter values MUST be compatible with input and output space dimensionality (see details).
+
+The following transformations are supported:
+
+| Type | Fields | Description |
+|------|--------|-------------|
+| `identity` | | The identity transformation is the default transformation and is typically not explicitly defined. |
+| `mapAxis` | `"mapAxis":List[number]` | A `mapAxis` transformation specifies an axis permutation as a transpose array of integer indices that refer to the ordering of the axes in the respective coordinate system. |
+| `translation` | one of:<br>`"translation":List[number]`,<br>`"path":str` | translation vector, stored either as a list of numbers (`"translation"`) or as binary data at a location in this container (`path`). |
+| `scale` | one of:<br>`"scale":List[number]`,<br>`"path":str` | scale vector, stored either as a list of numbers (`scale`) or as binary data at a location in this container (`path`). |
+| `affine` | one of:<br>`"affine": List[List[number]]`,<br>`"path":str` | affine transformation matrix stored as a flat array stored either with json using the affine field or as binary data at a location in this container (path). If both are present, the binary values at path should be used. |
+| `rotation` | one of:<br>`"rotation":List[List[number]]`,<br>`"path":str` | rotation transformation matrix stored as an array stored either with json or as binary data at a location in this container (path). If both are present, the binary parameters at path are used. |
+| `sequence` | `"transformations":List[Transformation]` | A sequence of transformations, Applying the sequence applies the composition of all transforms in the list, in order. |
+| `displacements` | `"path":str`<br>`"interpolation":str` | Displacement field transformation located at (path). |
+| `coordinates` | `"path":str`<br>`"interpolation":str` | Coordinate field transformation located at (path). |
+| `inverseOf` | `"transformation":Transformation` | The inverse of a transformation. Useful if a transform is not closed-form invertible. See Forward and inverse for details and examples. |
```

Comment:
What is "Forward and inverese"? It doesn't seem to be another section in this document.

Reply:
See a8f87138d8d2de5f5d982768f8cb2342c73f2cff